Makefile

```
@@ -71,6 +71,15 @@ $(REPOSITORIES):
 .PHONY: repos
 repos: $(REPOSITORIES)
 
+# Microkit
+
+MICROKIT_REPO = _repos/seL4/microkit
+_data/microkit_platforms.yml: $(MICROKIT_REPO)/platforms.yml
+	cp $(MICROKIT_REPO)/platforms.yml _data/microkit_platforms.yml
```

should we make tha variables so we don't have to repeat the file names?
```suggestion
	cp $< $@
```